src/views/mfa-verify/inputs/PasswordBox.js

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+    'okta',
+    'views/shared/TextBox'
+  ],
+  function (Okta, TextBox) {
+
+    return TextBox.extend({
```

Comment:
Couple nits I just noticed:
1. Probably should call this view something like "ShowPasswordBox" since that's the main reason you'd want to use it (a normal textBox is used for general passwords)
2. The spacing is off in this file (an extra set of two spaces throughout the entire file). Look at this file for reference on how we normally do it (specifically, the spacing around the modules for define, and where the function line sits):
https://github.com/okta/okta-signin-widget/blob/master/src/util/BaseLoginRouter.js